package.json

```diff
@@ -75,8 +75,8 @@
 		"webpack-merge": "^5.9.0"
 	},
 	"dependencies": {
-		"axios": "^1.7.2",
 		"@babel/runtime": "^7.22.15",
+		"axios": "^1.7.2",
```

Comment:
Seems like nothing changed here. Remove this change and the change in package-lock.json from the PR

Reply:
I installed Nodemailer and the dotenv package again, but it was just an updated version. Should I revert the packages to the old versions?

Reply:
This only change I can see in this file is the axios dependency being moved. I don't see any new dependencies.